types/react-fa/index.d.ts

```diff
@@ -37,7 +37,6 @@ export const Icon: ComponentClass<IconProps>;
 
 export interface IconStackProps extends SizeOverrideHTMLProps<IconStack> {
     size?: IconSize;
-    children?: IconProps[];
```

Comment:
This shouldn't be removed as it is required property (https://github.com/andreypopp/react-fa#iconstack-component-api). `children` should only contain element or elements of type `Icon` and thus you can improve the type to detect if that type has been passed. You can try something like:
```typescript
export interface IconNode<P> {
   type: ComponentClass<P & IconProps> | StatelessComponent<P & IconProps>
}

export interface IconStackProps extends SizeOverrideHTMLProps<IconStack> {
   size?: IconSize;
   children: IconNode<any> | Array<IconNode<any>>;
}
``` 
That's an example to you have to tweak it.